tests/bootstrap.php

```diff
@@ -0,0 +1,5 @@
+<?php
+namespace UNL\DWT;
+
+$autoloader = require __DIR__ . '/../vendor/autoload.php';
+$autoloader->add(__NAMESPACE__, __DIR__);
```

Comment:
This is why using the same namespace for tests get ugly. The autoloader will have duplicate entries for the same namespace. Composer supports adding `autoload-dev`: https://github.com/zendframework/zend-stdlib/blob/master/composer.json#L46 so no namespace nor autoloader adjustment should be necessary.